apps/desktop/src/routes/settings/experimental/+page.svelte

```diff
@@ -3,15 +3,14 @@
 	import {
 		featureBaseBranchSwitching,
 		featureInlineUnifiedDiffs,
-		featureBranchStacking,
+		stackingFeature,
```

Comment:
Just out of curiosity, why did you decide to go against the grain with this name change here? :sweat_smile: